server/routes/webmaker-app.js

```diff
@@ -69,6 +86,9 @@ module.exports = function (req, res, next) {
             res.send({
                 url: habitat.get('PUBLISH_URL') + '/' + dir
             });
+
+            tellIRC('publish', json);
```

Comment:
right now this only runs in the fake auth case, you will also want to run this in the real case when the s3 tasks have completed